spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
+  describe 'GET #index' do
+    it 'returns http success' do
+      get users_path
+      expect(response).to be_successful
+      expect(response.body).to include('List of users')
+    end
+  end
+
+  describe 'GET #show' do
+    let :user do
+      User.create do |user|
+        user.name = 'John Doe'
+      end
+    end
+
+    it 'returns http success' do
+      get user_path(user)
+      expect(response).to be_successful
+      expect(response.body).to include('User details')
+    end
+  end
```

Comment:
- [ ] Kindly make sure to test if the correct template was rendered in both the `show` and `index` actions, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=If%20a%20correct%20template%20was%20rendered.) requirement.